lib/surgex/repo_helpers/repo_helpers.ex

```diff
@@ -63,4 +67,20 @@ defmodule Surgex.RepoHelpers do
       opts
     end
   end
+
+  @doc """
+  Sets repo database ssl enable from specified env var.
+  """
+  def set_ssl(opts, env) do
+    cond do
+      String.downcase(System.get_env(env)) == "true" ->
+        Keyword.put(opts, :ssl, true)
+
+      String.downcase(System.get_env(env)) == "false" ->
+        Keyword.put(opts, :ssl, false)
+
+      true ->
+        Keyword.put(opts, :ssl, nil)
```

Comment:
Perhaps when ENV var is not present then we should not modify the setting.
```suggestion
      true -> opts
```